mlflow/xgboost.py

```diff
@@ -286,6 +287,30 @@ def autolog(importance_types=["weight"]):  # pylint: disable=W0102
     import xgboost
```

Comment:
Can we add docs to the `mlflow.xgboost.autolog()` method indicating that we also log the signature and input example when possible?